styles.css

```diff
@@ -0,0 +1,179 @@
+
+.tool-bar {
+  position: absolute;
+  width: 375px;
+  height: 92px;
+  left: 0;
+  top: 0;
+  background: #3c3a39;
+}
+
+.head-line {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  gap: 12px;
+  position: absolute;
+  width: 375px;
+  top: 92px;
+  background: #1c1a19;
+  background-image: url("img/Illustration.svg");
+  background-repeat: no-repeat;
+  background-position: bottom -50px right;
+
+  /* background: #1c1a19; */
+  height: 590px;
+  left: 0;
+}
```

Comment:
- [ ] Make sure you remove all commented code lines to make your codebase cleaner and more readable.

 - [ **OPTIONAL** ] **IMPORTANT:** To make your web page responsive I suggest you update the `width: 375px;` to be `  width: 100%;` and the `height: 590px` to be `height: 100vh`